src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/SOSLIIOFDOTAFE/XamlCodeGenerator_EmbeddedXamlSources.cs

```diff
@@ -0,0 +1,154 @@
+﻿#nullable enable
+// <autogenerated />
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+#pragma warning disable // Disable all warnings for this generated file
+
+// Register an embedded sources provider for Hot Reload
+[assembly: global::System.Reflection.AssemblyMetadata("Uno.HotDesign.HotReloadEmbeddedXamlSourceFilesProvider", "MyProject.__Sources__.EmbeddedXamlSourcesProvider")]
+
+namespace MyProject.__Sources__;
+
+/// <summary>
+/// Provides access to the embedded XAML sources
+/// </summary>
+/// <remarks>
+/// This class is used to provide the embedded XAML sources to the Hot Reload engine.
+/// This is not intended to be used directly by application code.
+/// WON'T BE GENERATED ON RELEASE BUILDS
+/// </remarks>
+internal static class EmbeddedXamlSourcesProvider
+{
+	// key=absolute file path
+	private static IDictionary<string, (string ActualPath, Func<(string Hash, string Payload)> Getter)>? _XamlSources;
```

Comment:
Better to generate it as `global::System.Collections.Generic.IDictionary<string, (string ActualPath, global::System.Func<(string Hash, string Payload)> Getter>`

Reply:
We'll make the changes in a separate PR